.travis.yml

```diff
@@ -14,16 +18,9 @@ addons:
       - libboost-date-time-dev 
       - libboost-test-dev
       - google-mock
- 
-before_script:
-  - "cd /usr/src/gmock && sudo cmake . && sudo cmake --build . && sudo mv libg* /usr/local/lib/ && sudo mv gtest/libg* /usr/local/lib/ ; cd -"
 
 script: ./travis.sh 
 
-branches:
-  only:
-    - master
-
```

Comment:
Yes, it is true that this limited the branches build on Travis to master only. To have other branches build (before creating a PR for instance), my method would have been that on that branch add that branch to the list of branches to build, and then remove it again before creating the PR.